src/stream.ts

```diff
@@ -148,8 +143,8 @@ export class WebRTCStream implements Stream {
       return;
     }
 
-    let self = this;
-    let closeWriteIterable = {
+    const self = this;
```

Comment:
don't we have linters that state something an be const given we do not write to it ever again?